### PR TITLE
new RSS feed

### PR DESCRIPTION
**Pull request by kyokukou**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ RUN pip install pipenv
 RUN apt-get update
 # Install python dependencies in /.venv
 COPY Pipfile .
-COPY Pipfile.lock .
+#COPY Pipfile.lock .
```
Comment (bdc34) [inline Dockerfile:17]: This is unexpected, usually we want the locked version of the libraries in the Docker image.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,56 +1,21 @@
-# Provides the Feed + ElasticSearch + Kibana stack for local development. See
-# README.md for instructions on use.
+# See README.md for instructions on use.
```
Comment (bdc34) [inline docker-compose.yml:1]: This docker-compose seems to only bring up one container. Maybe get rid of it if it does not seem useful?

File: feed/config.py [hunk 3]
```diff
@@ -18,31 +19,21 @@ class Config:
```
Comment (bdc34) [inline feed/config.py:19]: Maybe increment VERSION = "0.3" ?

File: feed/tests/test_utils.py [hunk 4]
```diff
@@ -40,3 +38,12 @@ def test_etag():
     assert etag("content") == etag(b"content")
     assert etag("Садржај") == etag("Садржај".encode("utf-8"))
     assert etag("foo") != etag("bar")
+
+
+@patch("feed.utils.datetime")
+def test_get_arxiv_midnight(mock_datetime, app):
+    mock_datetime.now.return_value = datetime(2023, 11, 10, 2, 30, 45, tzinfo=ZoneInfo('UTC'))
```
Comment (bdc34) [inline feed/tests/test_utils.py:45]: I like it.

**Review by bdc34 — APPROVED** [review]
This looks good to me.

File: feed/config.py (hunk 3 above)
Comment (kyokukou) [reply feed/config.py:19]: up to 0.4 or would this count as 1.0 since its theoretically getting released?

**Pull request merged**